controller/users.go

```diff
@@ -0,0 +1,57 @@
+package controller
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/go-chi/chi/v5"
+	"strconv"
+
+	"github.com/sirupsen/logrus"
+	"github.com/source-academy/stories-backend/model"
+)
+
+func GetUsers(w http.ResponseWriter, r *http.Request) {
+	users := model.GetAllUsers()
+
+	w.Header().Set("Content-Type", "application/json")
+	encoder := json.NewEncoder(w)
+	if err := encoder.Encode(&users); err != nil {
+		logrus.Errorln(err)
+		panic(err)
+	}
```

Comment:
This seems to be a lot of duplicated code in every single controller method. Perhaps we can abstract this into its own function?